pysrc/juliacall/__init__.py

```diff
@@ -109,9 +109,9 @@ def int_option(name, *, accept_auto=False, **kw):
         except ValueError:
             raise ValueError(f'{s}: expecting an int'+(' or auto' if accept_auto else ""))
 
-    def args_from_config():
-        argv = [CONFIG['exepath']]
-        for opt, val in CONFIG.items():
+    def args_from_config(config):
+        argv = [config['exepath']]
+        for opt, val in config.items():
```

Comment:
(This part just cleans up some spooky action at a distance I noticed in the init)